src/nomad_material_processing/__init__.py

```diff
@@ -62,21 +218,32 @@ def normalize(self, archive, logger: BoundLogger) -> None:
         super(Substrate, self).normalize(archive, logger)
 
 
+class CrystallineSubstrate(Substrate):
+    '''
+    The substrate defined in this class is composed of periodic arrangement of atoms
+    and shows typical features of a crystal structure.
+    '''
+    m_def = Section()
+        
+    geometry = SubSection(
+        section_def=Geometry
+    )
+    crystal_properties = SubSection(
+        section_def=SubstrateCrystalProperties
+    )
+    dopants = SubSection(
+        section_def=Dopant
+    )
+    
+    
 class ThinFilm(CompositeSystem):
```

Comment:
I would see it for example in SubstrateCrystalProperties, if it is only measured in a crystalline sample, otherwise somewhere ele in the substarte, maybe Geometry yes, we will discuss next round!